app/src/editor/canvas/components/ModuleSearch.jsx

```diff
@@ -24,31 +59,69 @@ export default class ModuleSearch extends React.PureComponent {
 
     async load() {
         const modules = await getModuleTree()
+        this.allModules = modules
```

Comment:
If `allModules` changes, shouldn't the UI update? i.e. why not `state.allModules`